CHANGELOG.md

```diff
@@ -46,6 +49,9 @@ along with the alphabet, were removed from the config.
 
 In SN configuration section `contracts` is gone and no longer needed.
 
+Experimental tree service is completely removed in this release, please drop
```

Comment:
i'd also suggest to rm corresponding DB files (iiuc they wont dissapear automatically)

Reply:
Makes sense.